hack/make-rules/build_with_container.sh

```diff
@@ -26,9 +26,17 @@ KUBEEDGE_BUILD_IMAGE=${KUBEEDGE_BUILD_IMAGE:-"kubeedge/build-tools"}
 DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
```

Comment:
```suggestion
DOCKER_GID="${DOCKER_GID:-$(grep '^docker:' /etc/group | cut -f3 -d:)}"
CONTAINER_RUN_OPTIONS="${CONTAINER_RUN_OPTIONS:-it}"
```
It's better to define a value :)